README.md

```diff
@@ -1737,41 +1690,41 @@ For example, instead of:
 ```php
 class ServiceA
 {
-    public function doSomething()
+    public function doSomething(): void
     {
-        $a = Manager::getInstance();
-        $a->doSomething('a');
+        $alpha = Manager::getInstance();
+        $alpha->doSomething('a');
     }
 }
 class ServiceB
 {
-    public function doSomething()
+    public function doSomething(): void
     {
-        $a = Manager::getInstance();
-        $a->doSomething('b');
+        $alpha = Manager::getInstance();
+        $alpha->doSomething('b');
     }
 }
-$a = new ServiceA();
-$b = new ServiceB();
+$alpha = new ServiceA();
```

Comment:
if we change a to Alpha, why do we keep the Service name as A? :) **ServiceAlpha**